### ISSUE TITLE
Accessibility - diagram text not announced to screen reader

### ISSUE THREAD

**Issue by elichad** [likec4/likec4#2988]
### Relates to

Diagrams (React)

### Version you are using

1.55.1

### Describe the issue

When browsing a LikeC4 diagram using the screen reader NVDA with Firefox (for example https://uomresearchit.github.io/5S-TES-diagram/view/index/, also true in the default playground example):
* none of the names/descriptions of nodes are announced. for every node NVDA announces "Press enter or space to select a node. You can then use the arrow keys to move the node around. Press delete to remove it and escape to cancel." (these controls also don't work, I assume they are for the edit mode of LikeC4)
* I can tab to a group of nodes but cannot descend into it to find out what is inside
* I cannot navigate to another view by selecting a node (as if I clicked on it)
* I cannot access the links between nodes to find out their descriptions or what they are connecting between

All of this makes LikeC4 diagrams effectively inaccessible to blind/visually impaired users who use screen readers.

### If possible, include link to the playground with example

https://uomresearchit.github.io/5S-TES-diagram/view/index/
